pyproject.toml

```diff
@@ -20,7 +20,7 @@ pre-commit-hooks = {git = "https://github.com/Lucas-C/pre-commit-hooks"}
 black = "*"
 hdwallet = "^1.3.2"
 ipython = "^7.28.0"
-slither-analyzer = "^0.8.3"
```

Comment:
Can be undone when 
- https://github.com/crytic/slither/issues/1228

is closed.